server/app/views/components/TextFormatter.java

```diff
@@ -108,9 +118,12 @@ private static String preserveEmptyLines(String text) {
 
   private static String addIconToLinks(String markdownText) {
     String closingATag = "</a>";
+
     String svgIconString =
         Icons.svg(Icons.OPEN_IN_NEW)
             .withClasses("shrink-0", "h-5", "w-auto", "inline", "ml-1", "align-text-top")
+            .attr("aria-label", ", " + ariaLabel)
+            .attr("aria-hidden", false)
```

Comment:
This is the change we talked about to add the "opens in a new tab" label to the svg.